### DISCUSSION TITLE
简报：哔哩哔哩相簿首页主图及部分功能已无法加载（附考古）

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4497] [综合讨论 / General]
![图片](https://github.com/the1812/Bilibili-Evolved/assets/81268244/ac8aa157-3d82-498f-b917-12336220d4cb)
首页只剩漆黑一片，一些 API 也返回了错误，可能离寿终正寝不远了

这是最后一个真正意义上的哔哩哔哩相簿投稿，标题、分区、标签什么的都有：https://h.bilibili.com/123424489 (2021-06-18)

之前也有由其他系统生成的投稿，不过现在就都是由其他系统生成的了

相簿系统（有标题、有分区）：
https://h.bilibili.com/1003 (2017-05-31) (疑似编号最小)
https://h.bilibili.com/3000 (2017-05-17)
https://h.bilibili.com/10000 (2017-03-18)
https://h.bilibili.com/19000 (2017-01-02)
https://h.bilibili.com/600001 (2014-11-04)
https://h.bilibili.com/620000 (2014-08-07)
https://h.bilibili.com/642000 (2014-07-08)
https://h.bilibili.com/647000 (2011-09-30)
https://h.bilibili.com/650001 (2011-07-25)
https://h.bilibili.com/652555 (2011-07-03) (疑似时间最早)

动态系统（无标题、默认日常分区）：
https://h.bilibili.com/50001 (2017-07-07)
https://h.bilibili.com/60111 (2017-07-11)
https://h.bilibili.com/457548 (2017-10-11)
https://h.bilibili.com/652601 (2017-10-13)
https://h.bilibili.com/2000000 (2018-02-03)
https://h.bilibili.com/10000000 (2018-12-07)
https://h.bilibili.com/30000000 (2019-07-31)
https://h.bilibili.com/50000000 (2020-01-19)
https://h.bilibili.com/123429000 (2023-04-04) 
https://h.bilibili.com/132433390 (2021-05-01)
https://h.bilibili.com/214864296 (2022-12-17)

工房系统：
https://h.bilibili.com/123430000 (2023-08-02)
https://h.bilibili.com/123930000 (2023-08-10)
https://h.bilibili.com/123960000 (2023-08-11)

也有 B 站测试人员的：
https://h.bilibili.com/60001 (2017-07-10)
https://h.bilibili.com/123427500 (2022-11-17)

这些编号真的非常混乱，不知道 B 站是怎么维护的

另外还发现了这个，曾经的哔哩哔哩有文画系统：
https://h.bilibili.com/ywh/h5/457548 会自动跳转至 https://t.bilibili.com/457548?tab=1&type=2
https://h.bilibili.com/ywh/457548 会自动跳转至 https://h.bilibili.com/457548

**Comment by the1812** [comment]
泪目

**Comment by Geeyun-JY3** [comment]
文画系统是什么

**Reply by Tinhone** [reply]
可能是相簿系统的前身，首页应该是 h.bilibili.com/ywh
另外全称是`哔哩哔哩有文画`（）

**Reply by Geeyun-JY3** [reply]
Google 了一下，「有文画」好像是个活动（：https://www.bilibili.com/blackboard/link-ywh.html

**Reply by Tinhone** [reply]
唔哦！不过如果是个活动的话，可能之后就不会出现 `h.bilibili.com/ywh/*` 这种链接，但不排除是自由的 B 站搞事（）
然后看起来`有文画`和`相簿`是一个系统的两个称呼？好奇怪喔
